### DISCUSSION TITLE
Difference between 11 and 11_374 Gerbers

### DISCUSSION THREAD

**Discussion by wrljet** [slabbi/Z80-CPU-Tester#2] [Q&A] [answered]
Can you explain what is the difference between `gerberz80testerv11.zip` and `gerberz80testerv11_374.zip`?

Thank you, Bill

**Comment by slabbi** [comment, marked as answer]
See note on this page:
https://github.com/slabbi/Z80-CPU-Tester/tree/main/Model%201/Hardware

**Comment by wrljet** [comment]
Oh, OK, now I understand.  I had read that, but not carefully enough to realize it was actually a slightly different circuit.
Thanks
